bundles/mapping/mapmodule/plugin/getinfo/GetFeatureInfoFormatter.js

```diff
@@ -125,6 +126,11 @@ Oskari.clazz.category('Oskari.mapframework.mapmodule.GetInfoPlugin', 'formatter'
             .forEach(frag => baseDiv.append(frag));
         return baseDiv;
     },
+
```

Comment:
Add description here what a formatter is and check that it has enabled and format functions before adding it.